rules.mk

```diff
@@ -26,7 +26,7 @@ cmd/boots/boots-linux-amd64: FLAGS=GOARCH=amd64
 cmd/boots/boots-linux-arm64: FLAGS=GOARCH=arm64
 cmd/boots/boots-linux-armv6: FLAGS=GOARCH=arm GOARM=6
 cmd/boots/boots-linux-armv7: FLAGS=GOARCH=arm GOARM=7
-cmd/boots/boots-linux-386 cmd/boots/boots-linux-amd64 cmd/boots/boots-linux-arm64 cmd/boots/boots-linux-armv6 cmd/boots/boots-linux-armv7: boots
+cmd/boots/boots-linux-386 cmd/boots/boots-linux-amd64 cmd/boots/boots-linux-arm64 cmd/boots/boots-linux-armv6 cmd/boots/boots-linux-armv7:
```

Comment:
the dep on `boots` here was to also depend on `boots`' deps, so maybe pull those out into a variable and add those as deps to these target and `boots` too?

Reply:
arrggg.....this was causing a double `go build` for me. Regardless, I've reverted this change. Feels out of scope for this PR anyway.